src/console_cmds.cpp

```diff
@@ -614,6 +615,115 @@ DEF_CONSOLE_CMD(ConBanList)
 	return true;
 }
 
+/**
+* Loads the ban list from the specified path.
+* @param path The path to load the ban list from.
+* @return The number of entries loaded from the file or -1 if an error occurred while loading entries.
+*/
+static int LoadBanList(const std::string& path)
```

Comment:
Coding style for OpenTTD dictates that the `&` is next to the variable/parameter.
The reason for this is that `int* p, q;` is equivalent to `int *p; int q;` and not `int* p; int* q;` even though someone would expect that given the "type" is `int*`.